client/src/Components/Common/Footer/style.css

```
@@ -0,0 +1,25 @@
+.footer  *{
+  background-color: var(--main-color) ;
+}
```

Here you can use id instead of class so it will be like that:
```suggestion
.#footer{
  background-color: var(--main-color) ;
}
```
and don't forget to change className to id in index.js